### ISSUE TITLE
Google search results include "Null safety" in the title

### ISSUE THREAD

**Issue by johnpryan** [dart-lang/dartdoc#2398]
When searching for a Flutter API, the "Null Safety" banner is included in the search result:

<img width="518" alt="Screen Shot 2020-10-19 at 5 19 07 PM" src="https://user-images.githubusercontent.com/1145719/96525343-817abe80-122f-11eb-99d7-1016c5a3438b.png">

The title of the page is correct:

```
<title>AnimationController class - animation library - Dart API</title>
```

But the text of the page uses a `<span>` tag:

```
<h1><span class="kind-class">AnimationController</span> class     <span class="feature feature-null-safety" title="Supports the null safety language feature.">Null safety</span>
 </h1>
```

**Comment by jcollins-g** [comment]
Hmm.  I know very little CSS/HTML, though I imagine the problem is that the null safety span is in the `<h1>` block.  Some other way of ensuring proper alignment with the heading will be needed.

This will impact other tags that go in this space, e.g. category.

**Comment by jcollins-g** [comment]
This appears to have been fixed accidentally, possibly by #2455.  The words "Null safety" no longer appear in the link title in google searches -- now they are back to the text, where they belong:  
![image](https://user-images.githubusercontent.com/14116827/108132301-76ae2a00-7067-11eb-843a-875a33bdf4a3.png)
